github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```diff
@@ -589,6 +591,12 @@ orgs:
         - victortoso
         repos:
           dra-pci-driver: admin
+      enhancements-maintainers:
+        maintainers:
+          - xpivarc
+          - vladikr
+        repos:
+          enhancements: admin
```

Comment:
nit: in the long run we should cut down on this, since it is not required most of the time. IMHO write access would be enough in general, or do you have something special in mind? Maybe you should consider tide merge automation.

However all that can be done in follow ups, will not block on this.

Reply:
@dhiller Thank you! 
I posted https://github.com/kubevirt/project-infra/pull/3876 to fix this.